File: c/fibo.c
#include <stdio.h>

int fibo(int num){
    if(num == 0 || num == 1){
        return num;
    }

    return fibo(num -1 ) + fibo(num -2);
}

int main(){
    int value = fibo(10);

    printf("value is: %i\n",value);
}

File: rust/fibo.rs
//fibo with recursion
fn fibo(n:i32) -> i32 {
    if n == 0 || n == 1 {
        return n;
    }
    
    return fibo(n - 1) + fibo(n - 2);
}

//fibo with loops


fn main(){
    //finding the fibonacci of any value 
    
    let value = fibo(10);
    
    println!("The fibo value of 10 is: {value}");

}